### PR TITLE
Some catwalk fixes

### LINKED ISSUE #6709: After catwalks get moved you can still place wires on their original tile
@paprka

### PR DESCRIPTION
**Pull request by ghost**
Fixes https://github.com/tgstation/-tg-station/issues/6709

Catwalks are now full tile due to a  visual bug with lattice smoothing, but they still look essentially the same and cleaner than bay/vg's ugly sprites.

Refactored cable placement a bit to apply to all turfs, not just simulated turfs, so you can put wires on walls or unsimulated turfs (if the turf's cancable var is 1, which it isn't by default on any of them)

Please priority review this, it's a pain to fix these map conflicts.

### CODE REVIEW

**Comment by pudl** [comment]
you should add alternative catwalk sprites (like floors have) since they're visually bland, imo

**Comment by paprka** [comment]
not in this pr

**Comment by Jordie0608** [comment]
>Even if there are 2 blobs the round will still end

I'm not so thrilled about that.

**Comment by Iamgoofball** [comment]
Instead of ending the round, either 1. make a random persom burst, or 2. spawn a new blob at the blob spawn marks

**Comment by Cheridan** [comment]
Yeah, picking a new candidate for blobbing sounds better than restarting the round out of the blue

**Comment by paprka** [comment]
Gee guys how would I go about that, feel free to provide all the code in question otherwise this solution is perfectly fine

Seriously I'm not letting this bug rot for 50000 more days, if you don't like my solution make a better one.

The round suddenly ending > an endless round because the shuttle can't be called and the blob is dead.

**Comment by AlexanderUlanH** [comment]
>paprka gets upset people are offering him suggestions again

If you remove the round-ending bit from your PR, I'll handle the monumental undertaking of making the blob autospawn replacements and message the admins about the suicide this afternoon.

**Comment by Kromgar** [comment]
Please let Alexander do what he said. I don't want Blob rounds ending because a retard went on solars because he didn't want to play blob

**Comment by paprka** [comment]
You guys can do whatever you want, if you make a fix that's better than my fix of course i'll remove it but i'm not holding my breath and i'm keeping it in until the PR is made because every time we do the YOU'RE FIX IS BAD I'LL MAKE A BETTER ONE, the new fix never gets made.

If someone goes onto the solars or space as a blob, fucking antagban them just like you would a wizard who kills himself every round?

**Comment by Anonus** [comment]
It'd be a good solution if blob was less rare as shit and you could get blob reliably, but one person walking into space like an idiot shouldn't ruin the round of someone they have no way to get into contact with before they walk into space.

**Comment by AlexanderUlanH** [comment]
#7440 is done, minus some final testing.  @paprka, you can remove the blob stuff from this PR.

**Comment by paprka** [comment]
As soon as it's accepted by maintainers sure

**Comment by paprka** [comment]
Removed the blob fixes in this PR

File: code/game/turfs/simulated/floor/plating.dm [hunk 1]
```diff
@@ -58,13 +58,6 @@
 			playsound(src, 'sound/weapons/Genhit.ogg', 50, 1)
 		else
 			user << "<span class='notice'>This section is too damaged to support a tile. Use a welder to fix the damage.</span>"
-	else if(istype(C, /obj/item/stack/cable_coil))
-		var/obj/item/stack/cable_coil/coil = C
-		for(var/obj/structure/cable/LC in src)
-			if((LC.d1==0)||(LC.d2==0))
-				LC.attackby(C,user)
-				return
-		coil.place_turf(src, user)
```
Comment (Razharas) [inline code/game/turfs/simulated/floor/plating.dm]: You removed this from attackby but didnt add ..() anywhere
0/5 coding

**Pull request merged**